src/services/entity-conversion.ts

```diff
@@ -44,7 +44,7 @@ export const convert = async <T> (tableName: string, sqlOperation: SqlOperation,
 };
 
 const getEntityConverter = <T> (tableName: string): EntityConverter<T> => {
-    if (tableName.includes("technical-records")) {
+    if (tableName.includes("technical-records") || tableName.includes("flat-tech-records")) {
```

Comment:
Is this right? Is this the destination table in NOP?

Reply:
tableName is retrieved from the inbound event's eventSourcARN, it's not a reference to the table in NOP

Reply:
Thanks for explaining :)